src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/SettingSelector.cs

```diff
@@ -24,7 +30,7 @@ public class SettingSelector
         /// <remarks>See the documentation for this client library for details on the format of filter expressions</remarks>
         public IList<string> Labels { get; set; }
         /// <summary>
-        /// IKeyValue fields that will be retrieved.
+        /// Specify which fields will be retrieved.
```

Comment:
To me, the phrasing feels a bit awkward here.   Maybe we should consider something like `Controls the fields that will be populated` or `Allows requesting a specific set of fields` ?